src/applications/disability-benefits/526EZ/components/ButtonContainer.jsx

```diff
@@ -66,7 +67,13 @@ export default function ButtonContainer(props) {
         </a>
       )}
       {!atGuidance() && (
-        <a className="usa-button-primary" onClick={goForward}>
+        <a
```

Comment:
I'm not sure why this was an anchor in the first place. Since it doesn't have an `href` and is _trying_ to act like a button, we should use the semantic `<button>` element.

I tested the following and it worked as expected.
```jsx
        <button className="usa-button-primary" onClick={goForward}>
          Next
          <span className="button-icon"> »</span>
        </button>
```

Reply:
I agree with this ^

Reply:
Changed to button. Thank you!